cmd/core/main.go

```diff
@@ -164,6 +166,23 @@ func main() {
 		}
 	}
 
+	switch strings.ToLower(applyOnceOnly) {
+	case "", "false", string(oamcontroller.ApplyOnceOnlyOff):
+		controllerArgs.ApplyMode = oamcontroller.ApplyOnceOnlyOff
+		setupLog.Info("ApplyOnceOnly is disabled")
+	case "true", string(oamcontroller.ApplyOnceOnlyOn):
+		controllerArgs.ApplyMode = oamcontroller.ApplyOnceOnlyOn
+		setupLog.Info("ApplyOnceOnly is enabled, that means workload or trait only apply once if no spec change even they are changed by others")
+	case string(oamcontroller.ApplyOnceOnlyForce):
+		controllerArgs.ApplyMode = oamcontroller.ApplyOnceOnlyForce
+		setupLog.Info("ApplyOnceOnlyForce is enabled, that means workload or trait only apply once if no spec change even they are changed or deleted by others")
+	default:
+		setupLog.Error(fmt.Errorf("invalid apply-once-only value: %s", applyOnceOnly),
+			"unable to setup the vela core controller",
+			"valid apply-once-only value:", "true/false/on/off/force ")
```

Comment:
```suggestion
			"valid apply-once-only value:", "on/off/force ")
```

true/false is for compatibility, don't need to tell